mne/minimum_norm/tests/test_inverse.py

```diff
@@ -389,9 +389,11 @@ def test_make_inverse_operator_vector():
     inv_2 = make_inverse_operator(evoked.info, fwd_surf, noise_cov, depth=None)
     inv_3 = make_inverse_operator(evoked.info, fwd_surf, noise_cov, fixed=True,
                                   loose=None)
+    inv_4 = make_inverse_operator(evoked.info, fwd_fixed, noise_cov,
+                                  loose=.2, depth=None)
 
     # Apply the inverse operators and check the result
-    for inv in [inv_1, inv_2]:
+    for inv in [inv_1, inv_2, inv_4]:
         for method in ['MNE', 'dSPM', 'sLORETA']:
```

Comment:
you probably don't need to test MNE, dSPM and sLORETA here.

I want to limit the test time